src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.wasm.cs

```diff
@@ -44,6 +44,18 @@ private void UpdateTextBoxView()
 			}
 		}
 
+		partial void ValidateTextRange()
+		{
```

Comment:
It's confusing to have a method called `Validate...()` raising events or having side effects like reseting the caret...

Reply:
Thank you for your comment, what do you suggest to clarify? When is not a valid range,  SelectionChanged fired to do the normal flow and not create side-effects.